website/content/docs/agent-and-proxy/agent/index.mdx

```diff
@@ -140,6 +142,10 @@ These are the currently-available general configuration options:
 
 - `template_config` <code>([template_config][template-config]: <optional\>)</code> - Specifies templating engine behavior.
 
+- `exec` <code>([exec][exec]: <optional\>)</code> - Specifies options for vault agent to run a child process that injects secrets (via `env_template` stanzas) as environment variables
+
+- `env_template` <code>([env_template][template]: <optional\>)</code> - Multiple blocks accepted. Each block contains the options used for templating Vault secrets as environment variables via exec
```

Comment:
I think it's worth noting the linkage between these two stanzas, i.e. you can't have one without the other, and you can't have these with the other subsystems